docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -0,0 +1,13 @@
+=======================
+Pulp 2.11 Release Notes
+=======================
+
+Pulp 2.11.0
+===========
+
+New Features
+------------
+
+* Re-sync of the repository will be a no operation by default if since the last sync, no units
```

Comment:
Would you say that since there is no change in behavior from a user perspective (logic has simply been moved rather than added, necessarily), it doesn't need a release note at all?

I agree that it's incorrect as it exists currently - I can fix it if there does need to be a note.